lib/autoload/filename_sanitizer.rb

```diff
@@ -0,0 +1,8 @@
+module FilenameSanitizer
+  # Sanitizes the filename.
+  #
+  # @param [String] filename
+  def sanitize(filename)
+    filename.gsub(/[:\/\\]/, ' ')
```

Comment:
Use `%r` around regular expression.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.